regression/verilog/functioncall/functioncall2.desc

```diff
@@ -3,6 +3,6 @@ functioncall2.v
 --module main --bound 1
 ^EXIT=0$
 ^SIGNAL=0$
-^UNSAT: No counterexample found within bound$
+^UNSAT: .*$
```

Comment:
Can we use this (or any other) test to exercise bmc-with-assumptions?

Reply:
Test added